personalization-webcomponents/src/checklist-detail.ce.vue

```diff
@@ -4,31 +4,42 @@
     <div v-html="mucIconsSprite" />
     <!-- eslint-disable-next-line vue/no-v-html -->
     <div v-html="customIconsSprite" />
-    <muc-callout>
-      <template #header>
-        <p>checklist-detail Webcomponent</p>
-      </template>
-      <template #content>
-        <p>{{ calloutContent }}</p>
-      </template>
-    </muc-callout>
+
+      <div v-if="loading">
+        <skeleton-loader/>
+      </div>
+      <div v-else>
+        <checklist-header
+            :checklist="checklists[0]"
+        >
+        </checklist-header>
+      </div>
   </div>
 </template>
 
 <script setup lang="ts">
-import { MucCallout } from "@muenchen/muc-patternlab-vue";
 import customIconsSprite from "@muenchen/muc-patternlab-vue/assets/icons/custom-icons.svg?raw";
 import mucIconsSprite from "@muenchen/muc-patternlab-vue/assets/icons/muc-icons.svg?raw";
-import { computed } from "vue";
+import {onMounted, ref} from "vue";
+
 
-import { FIRSTNAME_DEFAULT } from "@/util/constants";
+import ChecklistHeader from "@/components/ChecklistHeader.vue";
+import type DummyChecklist from "@/api/dummyservice/DummyChecklist.ts";
+import DummyChecklistService from "@/api/dummyservice/DummyChecklistService.ts";
+import SkeletonLoader from "@/components/common/skeleton-loader.vue";
 
-const { firstName = FIRSTNAME_DEFAULT } = defineProps<{
-  firstName?: string;
-}>();
+const checklists = ref<DummyChecklist[]>([]);
```

Comment:
Because of the wrong api usage below this should become a single DummyCHecklist and not an array as this component should display only one checklist